package.json

```diff
@@ -48,10 +64,69 @@
   "scripts": {
     "postpack": "rm -f oclif.manifest.json",
     "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
-    "test": "echo NO TESTS",
+    "test": "jest --runInBand",
```

Comment:
is `runInBand` required? we typically add it for tests that interact with a db, but otherwise parallel should be 👍

Reply:
I had that in there previously because I needed the tests to run in a certain order, otherwise they failed. I refactored my code so that it didn't require that (which was better anyway). Now it seems to be working so I'll remove 👍

Reply:
🕺

Reply:
Sorry, I realized we actually do need this. We can't have the tests run in parallel because they modify files also used by the other. Both the `init` and `generate` commands use/adapt the `package.json` during their tests. We restore everything back to normal (because we make copies), but because of this they **cannot run in parallel**.